### DISCUSSION TITLE
Additional examples?

### DISCUSSION THREAD

**Discussion by holdupmaster** [Bodmer/TFT_eSPI#3405] [Q&A - General]
Have been converting some examples of peoples code, where they typically use the Adafruit libraries - to use the TFT_eSPI library.  Performance difference is typically immediately noticeable.  Is there a way I can share them in the example repo?

**Comment by Bodmer** [comment]
Create your own repo, drop them there and I will add a link to it.

**Comment by holdupmaster** [comment]
will do thanks!
